socialist_ir/in_lam_01_stack.py

```diff
@@ -0,0 +1,59 @@
+from PyInquirer import prompt
+from socialist_ir.cdk_menu import StackMenu
+from socialist_ir.utils.validators import (
+    SlackWebhookValidator,
+)
+from socialist_ir.config import Config
+
+
+class InLam01Stack(StackMenu):
+    def __init__(
+        self,
+        name: str = "in-lam-01-stack",
+        required_variables: list = ["webhook_url"],
+    ):
+        super().__init__(name=name, required_variables=required_variables)
+
+    '''
+    def test(self) -> None:
```

Comment:
Any automated test?

Reply:
No automated test as it will deny a bunch of permissions on the user who deployed it - Should be pretty easy to code one up though; Is there a way to show warnings to user that this could lock their IAM permissions?

Reply:
Yeah, you can use PyInquirer to create a y/n menu